intelmq/bots/outputs/file/output.py

```diff
@@ -14,12 +15,21 @@ def init(self):
 
     def process(self):
         event = self.receive_message()
+        
 
         event_data = event.to_json(hierarchical=self.parameters.hierarchical_output)
-        self.file.write(event_data)
-        self.file.write("\n")
-        self.file.flush()
-        self.acknowledge_message()
+
+        try:
+            os.stat(self.parameters.file)
+        except FileNotFoundError:
+            self.logger.debug("Reopening %r file." % self.parameters.file)
+            self.file = io.open(self.parameters.file, mode='at', encoding="utf-8")
+            self.logger.info("File %r is open." % self.parameters.file)
```

Comment:
The very same line in init() also uses INFO-level.

These 3 lines could be replaced by simply calling `self.init()`.